### ISSUE TITLE
when following privately: give user more info

### ISSUE THREAD

**Issue by Erkan-Yilmaz** [miguelfreitas/twister-html#303]
e.g. 
1 consequence: info loss at browser exit (1)

here a report by a new user:
"Following people privately appears to not work.Subscriptions are lost after process restart.Only public following works.Known issue?"

(1) since the private subscriptions stores in local storage of your browser, when browser option to delete info (cookies,...) at exit

**Comment by miguelfreitas** [comment]
yes, twisterd may include a way to persist this public/private following information. I think it makes sense.
